docs/source/user-guide/overview.md

```diff
@@ -29,7 +29,7 @@ Comet aims to support:
 - a native Parquet implementation, including both reader and writer
 - full implementation of Spark operators, including
   Filter/Project/Aggregation/Join/Exchange etc.
-- full implementation of Spark built-in expressions
+- [full implementation](../../../docs/spark_expressions_support.md) of Spark built-in expressions.
```

Comment:
This won't build correctly:


```
/Users/andy/git/apache/datafusion-comet/docs/temp/user-guide/overview.md:32: WARNING: Unknown source document '../spark_expressions_support' [myst.xref_missing]
```

Let's revert this change for this PR and handle where we publish (user guide vs contributor guide) in a follow-up PR.